sass/components/Wizard/create-vm-wizard.scss

```diff
@@ -14,6 +14,11 @@
   text-align: left;
 }
 
+.kubevirt-create-vm-wizard__result-tab-row--error {
+  background-color: mistyrose;
+  border-color: red;
```

Comment:
can we use #c9190b here to be consistent.

Reply:
done